source/core/TypeAndDataReflection.cpp

```diff
@@ -27,13 +27,13 @@ namespace Vireo
 
 class DataReflectionVisitor : public TypeVisitor
 {
-public:
```

Comment:
Do we want a single space or four spaces? if we want four spaces then we need to move most of the file contents to the right.